### ISSUE TITLE
pull request

### ISSUE THREAD

**Issue by mdjames094** [kirkthaker/investopedia-trading-api#19]
Dear kirkthaker. Please have a look at my pull request. best regards.

**Comment by kirkthaker** [comment]
Thanks for the pull request! I'm traveling this week, so I'll probably take a good look at it this weekend.

**Comment by MSXR** [comment]
is this valid use of issue section? Just wondering.

**Comment by kirkthaker** [comment]
Yep, it's fine. School has taken up way more time than expected so I haven't had as much time for side projects as I would like. I'll get to this eventually, I promise!

**Comment by Jvinniec** [comment]
@kirkthaker Seeing 3 pull requests that have not been addressed for 4 months makes it seem like this project has reached end-of-life, which I'm sure you don't intend. Given the popularity of this repository (90 stars and 30 forks is certainly more than anything I've ever personally developed), I'm curious if you have considered adding a collaborator or two? People who could step up and make pull request decisions on your behalf. Obviously you would want to choose people who you are comfortable giving control of the repository to, but it would reduce the amount of pressure on your end and keep the project from looking like it's not being supported any more. 

Thoughts?

**Comment by kirkthaker** [comment]
I agree - adding some collaborators would keep to project moving forward. I would love to add a collaborator or two so that the pull requests could be merged without my involvement. However, I'm not sure where I would find collaborators?

Anyone interested in being a collaborator (mostly just merging pull requests and keeping the project alive) can comment below this and LMK, and I'd be happy to add them. I might also make a post in a subreddit if we don't get enough people this way.

**Comment by josiahdahl** [comment]
@kirkthaker I'd be interesting in helping to maintain.

**Comment by kirkthaker** [comment]
@josiahdahl I've added you as a collaborator - let me know if you need more permissions. Thanks for helping - I really appreciate it!

**Comment by josiahdahl** [comment]
@kirkthaker Awesome :) Thanks for merging those PRs as well.
